main/webapp/modules/core/langs/translation-en.json

```diff
@@ -364,12 +364,12 @@
     "core-project/remove-all": "Remove all facets",
     "core-project/export-project": "Export project",
     "core-project/project-data-package": "Project data package",
-    "core-project/tab-value": "Tab-separated value",
-    "core-project/comma-sep": "Comma-separated value",
-    "core-project/html-table": "HTML table",
-    "core-project/excel": "Excel (.xls)",
-    "core-project/excel-xml": "Excel 2007+ (.xlsx)",
-    "core-project/odf": "ODF spreadsheet",
+    "core-project/tab-value": "Tab-separated value…",
+    "core-project/comma-sep": "Comma-separated value…",
+    "core-project/html-table": "HTML table…",
+    "core-project/excel": "Excel (.xls)…",
```

Comment:
For exporters which do not have any dialog associated with them, I would leave them without "…" (not all browsers create a download dialog, and I would say that does not really count anyway since it does not actually offer any "logical" options)

Reply:
I wasn't sure about the export actions. In my Brave browser on Windows 10, I get a download dialog prompting for a filename. After you commented, I tried Chrome, Firefox, and Edge and those browsers immediately download the file with the default name e.g. birds-csv.csv. Since clicking the options to export and download does so without further input in most browsers, the ellipsis is not necessary.

Ellipsis have been removed for the following actions:
OpenRefine project archive to file...
Tab-separated value...
Comma-separated value...
HTML table...
Excel (.xls)..
Excel 2007+ (.xlsx)...
ODF spreadsheet...